src/devices/bus/coco/cococart.cpp

```diff
@@ -443,6 +445,21 @@ device_cococart_interface::~device_cococart_interface()
 
 
 //-------------------------------------------------
+//  interface_config_complete
+//-------------------------------------------------
+
+void device_cococart_interface::interface_config_complete()
+{
+	m_owning_slot = dynamic_cast<cococart_slot_device *>(device().owner());
+	if (!m_owning_slot)
```

Comment:
You can't safely do this check at config complete time, you need to move it to start time.  The reason is that the auxiliary verbs instantiate devices inside an empty machine, and in that case the owner will be some kind of `driver_device`.  They have no way to know what kind of structure the devices expect around them, so they can't build the expected relationships.  So you can do the dynamic casts here, but you need to move the checks for null to `interface_pre_start`.